Plots/Bar/NCL_bar_7.py

```diff
@@ -93,7 +93,7 @@
 # Add the legend
 plt.legend(bars_reversed, labels_reversed)
 
-# Use geocat.viz.util convenience function to set axes limits & tick values without calling several matplotlib functions
+# Use geocat.viz.util convenience function to set axis limits & tick values without calling several matplotlib functions
```

Comment:
```suggestion
# Use geocat.viz.util convenience function to set the axes limits & tick values without calling several matplotlib functions
```
This will also be axes because it is applying to both axes at once